src/testing/mocking.md

```diff
@@ -0,0 +1,58 @@
+---
+minutes: 10
```

Comment:
I don't want to litigate down to the minute, IMHO this slide should be short: "If it's mocking you need, there's a great crate for that which is used in Android, called mockall. Here's an example, which creates a MockPet type. Find more at docs.rs when you need it"

Reply:
Yeah, maybe 10 minutes is overestimating it here. I really have to schedule a class myself to understand how much time to spend on each slide.

Let change it to 5 minutes as a start?

```suggestion
minutes: 5
```